<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>download_complete</class>
 <widget class="QWidget" name="download_complete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Completed</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon/logo.png</normaloff>:/icon/icon/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>7</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icon/icon/folder.png&quot; width=&quot;81&quot; height=&quot;81&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Completed</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size_of_downlaod">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>501</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Downloaded </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Address :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lbl_url">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>591</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>The file saved as :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lbl_saved_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>591</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_folder">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
